### ISSUE TITLE
Look into creating usage metric dashboard

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#243]

**Comment by bhack** [comment]
What is this?

**Comment by bhack** [comment]
Do you mean something like https://github.com/tensorflow/addons/issues/236?

**Comment by LukeWood** [comment]
Just a script that collects metrics like pypi downloads and such every few days on a cron job.  mainly for my own usage

**Comment by bhack** [comment]
Ah ok, so it is much simpler.

If you could find it useful these are my two boards on keras-cv:
https://cauldron.io/project/5989
https://pypistats.org/packages/keras-cv

**Comment by bhack** [comment]
If you like badges:
Keras-cv Pipy stats (other configurations/customization at at https://shields.io/category/downloads)

https://img.shields.io/pypi/dm/keras-cv.svg
![image](https://img.shields.io/pypi/dm/keras-cv.svg)

Cauldron
https://cauldron.io/project/5989/stats.svg
![image](https://cauldron.io/project/5989/stats.svg)

**Comment by LukeWood** [comment]
Thanks for the idea - I added the downloads badge!  Thats really cool!

**Comment by LukeWood** [comment]
The KerasCV team is attempting to narrow the package scope in order to more efficiently use our limited resources. As such, we are only accepting contributions directly related to items on the 6 month roadmap for the package.

If you feel this feature could be particularly useful, or would like to contribute it - please feel free to re-open this issue and ping LukeWood, ianstenbit or tanzhenyu

Feel free to read more on the roadmap:
https://github.com/keras-team/keras-cv/blob/master/.github/ROADMAP.md
